projects/cdk/directives/active-zone/active-zone.directive.ts

```diff
@@ -76,7 +76,7 @@ export class TuiActiveZoneDirective implements OnDestroy {
     }
 
     @tuiPure
-    private setZone(zone: TuiActiveZoneDirective | null) {
+    private setZone(zone: TuiActiveZoneDirective | null | undefined) {
```

Comment:
Still some `undefined` left here and above :)

Reply:
fixed